julia_src/Dockerfile

```diff
@@ -24,6 +24,10 @@ ARG XPRESS_LICENSE_HOST=licenseserver
 RUN set -x && printf "f\nc\n\nn\n\n${XPRESS_LICENSE_HOST}\ny\n" | ./install.sh >> license_info.txt;
 RUN rm xp8.12.3_linux_x86_64.tar.gz
 
+# Install gfortran for /ghpghx
+RUN apt-get update
```

Comment:
This line and the next should be one `RUN` command (see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run).

Reply:
Done